bundle/manifests/ibm-websphere-liberty.clusterserviceversion.yaml

```diff
@@ -570,6 +570,10 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
+                - name: POD_NAMESPACE
```

Comment:
The existing environment variable OPERATOR_NAMESPACE should be used instead of creating a new one. The util method GetOperatorNamespace in RCO ([here](https://github.com/application-stacks/runtime-component-operator/blob/4e0aef35cc265f473ffe286a0abb7c0f37d75b08/utils/utils.go#L1192)) should be fixed to look up the right env variable (OPERATOR_NAMESPACE). Once the changes are delivered to RCO, the go module in WLO should be updated to pick up the changes from RCO.

Reply:
@leochr  https://github.com/application-stacks/runtime-component-operator/pull/383